db/migrate/20200901121606_add_constraints_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddConstraintsToUsers < ActiveRecord::Migration[5.2]
+  def change
+  	change_column :users, :username, :string, null: true, default: ""
+  end
+end
```

Comment:
Merge all three `AddConstraintsToUsers` files into one. Also always name each migration different. Though in this case you only need 1 file for all 3.